src/SharpLearning.Backend.Cntk/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="CNTK.GPU" version="2.2.0" targetFramework="net461" />
```

Comment:
can we support both GPU and CPU by using both packages? would we need two projects for that? Does CNTK not have a separate package independent of device?

Reply:
The CNTK.GPU package supports both CPU and GPU, while the CNTK.CPUOnly, only supports CPU. As far as I know, the CPUOnly package was only made in order to minimize the size of the package, avoiding all the GPU dependencies, for CPU only uses. So we should be able to support both devices only using the GPU package.

This might be different with tensorflow, which to my knowledge has a separate build for CPU and GPU.

Reply:
Ok, we would have the same issue then, if we wanted to minimize size and only want run CPU only, it is a bit unfortunate that "runtime" hasn't been separated from "type definitions/API", so only the project that executes the code would depend on the "runtime". Anyway, for now focus is on training so we have to live with it, but we should keep it in mind when defining types.